recipe/conda_build_config.yaml

```diff
@@ -4,7 +4,3 @@ MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
   - "10.14"                # [osx and x86_64]
 CONDA_BUILD_SYSROOT:       # [osx and x86_64]
   - /opt/MacOSX10.10.sdk   # [osx and x86_64]
-c_compiler:    # [win]
```

Comment:
Was this removed because it's outdated, or for another reason?

Reply:
yeah, it's in the base cbc.yaml now.

Reply:
Yup, it's in the base `cbc.yaml`